services/instagram/instagram.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const instagramSchema = Joi.object({
+  graphql: Joi.object({
+    user: Joi.object({
+      edge_followed_by: Joi.object({
+        count: Joi.number()
```

Comment:
Could the `nonNegativeInteger` helper be used here?

Reply:
Good idea!